wiki/Community/Video_series/lazer_updates/en.md

```diff
@@ -0,0 +1,64 @@
+| 21 | [lazer updates - June 21, 2023](https://www.youtube.com/watch?v=QMBmLI8BFcw) | Follow-ups to the implementation of the new scoring system, the Synesthesia mod, adjustments to the "argon" skin, [grid snapping](/wiki/Beatmapping/Grid_snapping) for osu!catch, [time-distance equality](/wiki/Beatmapping/Mapping_techniques/Time-distance_equality) principle in the editor, song selection | [(↗)](https://osu.ppy.sh/home/news/2023-06-22-osulazer-updates-june-21-2023) |
+| 22 | [lazer updates - July 16, 2023](https://www.youtube.com/watch?v=RJwK01SN2lg) | How [accuracy](/wiki/Gameplay/Accuracy) and [combo](/wiki/Beatmapping/Combo) affect scores, in-game hotkeys, new "argon" key counter, [stacking](/wiki/Beatmapping/Mapping_techniques/Stack) between objects, new osu!taiko hitsounds, built-in tablet driver, [osu!framework](https://github.com/ppy/osu-framework) | [(↗)](https://osu.ppy.sh/home/news/2023-07-16-osulazer-updates-july-16-2023) |
+| 23 | [lazer updates - August 11, 2023](https://www.youtube.com/watch?v=JLqhjygGDYI) | The [tournament client](/wiki/osu!_tournament_client), adjustment for legacy skins, clocks, various editor improvements, backward compatibility for osu!(lazer) beatmaps, user interface improvements | [(↗)](https://osu.ppy.sh/home/news/2023-08-11-osulazer-updates-august-11-2023) |
+| 24 | [path to lazer ranked play](https://www.youtube.com/watch?v=crkT0aaowKQ) | The state of play in osu!(lazer), [game infrastructure](https://github.com/ppy/osu-infrastructure), game balance, [hit window mechanics](/wiki/Gameplay/Judgement/osu!mania) in osu!mania, general user experience, performance optimizations | [(↗)](https://osu.ppy.sh/home/news/2023-09-08-osulazer-updates-the-path-to-ranked-play) |
+| 25 | [lazer updates - September 24, 2023](https://www.youtube.com/watch?v=SWEE8XlGluI) | Issues yet to be resolved for ranked play in osu!(lazer), [sliderhead](/wiki/Gameplay/Hit_object/Slider/Sliderhead) behaviour, mod select screen, various bug fixes | [(↗)](https://osu.ppy.sh/home/news/2023-09-25-osulazer-updates-september-24) |
+| 26 | [lazer updates - October 8, 2023](https://www.youtube.com/watch?v=pspL5SLs80k) | Rate adjustments for the Double Time and Half Time mod, spinner mechanics, new elements for the "argon" skin, legacy scoring systems, performance optimizations | [(↗)](https://osu.ppy.sh/home/news/2023-10-08-osulazer-updates-october-8) |
+| 27 | [lazer updates (and some hidden mechanics) - October 26, 2023](https://www.youtube.com/watch?v=SlWKKA-ltZY) | [Slider leniency](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners), [floating point](https://en.wikipedia.org/wiki/Floating-point_arithmetic) calculation, spinner overhaul, [grid snapping](/wiki/Beatmapping/Grid_snapping) for osu!mania and osu!taiko | [(↗)](https://osu.ppy.sh/home/news/2023-10-26-osulazer-updates-october-26) |
```

Comment:
it's something completely different from the regular grid snap (that's focused on visual object placement), this is a grid for actual snaps, and it already existed in mania prior.

opted for the wording from the news post.

```suggestion
| 27 | [lazer updates (and some hidden mechanics) - October 26, 2023](https://www.youtube.com/watch?v=SlWKKA-ltZY) | [Slider leniency](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners), [floating point](https://en.wikipedia.org/wiki/Floating-point_arithmetic) calculation, spinner overhaul, beat snap grid for the osu!taiko editor | [(↗)](https://osu.ppy.sh/home/news/2023-10-26-osulazer-updates-october-26) |
```